testing/mockClient.ts

```diff
@@ -0,0 +1,51 @@
+const mockClient = {
```

Comment:
Think we should extract this back into the app SDK

Reply:
done https://github.com/DeskproApps/app-testing-utils